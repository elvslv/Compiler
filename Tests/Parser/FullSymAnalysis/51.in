var i, j, k, f: integer;
r: real;
procedure proc(m, n: integer);
begin
	m := 10;
end;
begin
	if (proc(10, 9))
		i := 10;
end.