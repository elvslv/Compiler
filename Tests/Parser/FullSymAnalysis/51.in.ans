var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
var F : INTEGER;
var R : REAL;

procedure PROC(var M : INTEGER;var N : INTEGER;);

begin     
|         
|___:=    
|   |     
|   |___M 
|   |     
|   |___10
|         
end       


20 8 Variable must be typed