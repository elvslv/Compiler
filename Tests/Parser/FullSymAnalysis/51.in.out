var F : INTEGER;
var I : INTEGER;
var J : INTEGER;
var K : INTEGER;
procedure PROC(var M : INTEGER;var N : INTEGER;);
begin     
|         
|___:=    
|   |     
|   |___M 
|   |     
|   |___10
|         
end       

var R : REAL;

20 8 Variable must be typed